google-sheets/functions/check-sheets-config.js

```diff
@@ -0,0 +1,48 @@
+const { google } = require('googleapis');
+const path = require('path');
+
+exports.handler = async function(context, _event, callback) {
+    const response = new Twilio.Response();
+    response.appendHeader('Content-Type', 'application/json');
+
+    try {
+        const authJson = require(path.join('..', context.SHEETS_AUTH_JSON));
+        const auth = new google.auth.JWT({
+            email: authJson.client_email,
```

Comment:
i wonder if we should be adding a check for `authJson.client_email` and `authJson.private_key` to exist before making this call. Writing out an error message if this is false

Reply:
Added, plus a couple of extra checks to ensure the JSON file isn't a skeleton.